CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Mapbox welcomes participation and contributions from everyone.
 * Use a single shared buffer across all globe tiles to increase globe rendering performance. ([#1351](https://github.com/mapbox/mapbox-maps-android/pull/1351))
 * Re-introduce partial tile loading feature that decreases map load times. ([#1351](https://github.com/mapbox/mapbox-maps-android/pull/1351))
 * The `TilesetDescriptorOptions.pixelRatio` parameter is now passed to the TileStore and considered for the raster tile pack loading. This enables loading of a raster tilepacks for retina displays. ([#1351](https://github.com/mapbox/mapbox-maps-android/pull/1351))
+* Introduce earlyStyleCallback parameter to loadStyle APIs. ([#1353](https://github.com/mapbox/mapbox-maps-android/pull/1353))
```

Comment:
I wonder how we should clearly state to users what _could_ be run in earlyStyleCallback and what could not. For now early style callback will fit for projection and 3D modeling but will not work with runtime styling fully

Reply:
yeah, I've added detailed documentation to this flag below:
```
@param earlyStyleCallback whether to invoke [onStyleLoaded] callback earlier on the `StyleDataLoaded#Style`]
   *    event, default to false. Enable earlyStyleCallback helps with getting the Style Object faster,
   *    so that the runtime styling can be applied immediately after the style is parsed.
```
However, probably we should also state the limitations where this flag shouldn't be used, e.g. when the runtime styling is using the sources from the style url.